tests/test_query_parsing.py

```diff
@@ -11,25 +11,40 @@
 
 DB_URL = "sqlite+aiosqlite://"
 
+QUERY_STR = 'query_str'
```

Comment:
Add comments to separate different parts' parameters.Something like 'normal query extraction parameters', 'udf query extraction parameters'.